WalletCore/Classes/ViewModel/CashOut/CashOutViewModel.swift

```diff
@@ -38,6 +38,8 @@ public class CashOutViewModel {
     
     private let disposeBag = DisposeBag()
     
+    private let countryCodes = Variable<[LWCountryModel]>([])
```

Comment:
Do you need ViewModel's property countryCodes.Does not look that it's used within the ViewModel itself ?